src/proteus/interior/spider.py

```diff
@@ -376,13 +478,12 @@ def ReadSPIDER(dirs:dict, config:Config, R_int:float):
     output["RF_depth"]        = float(data_a[7])/R_int  # depth of rheological front
 
     # Manually calculate heat flux at near-surface from energy gradient
-    json_file   = MyJSON( dirs["output"]+'/data/{}.json'.format(sim_time) )
```

Comment:
Where did this go?

Reply:
The code itself is still there - I just commented it out because we weren't actually using the values for anything. I left it in because it's useful for testing things sometimes, but there's no point using the extra resources to read the file if the values don't go anywhere.